### PR TITLE
1.2 rc - japanese translation

### PR DESCRIPTION
**Pull request by coky-t**
Thank you for submitting a Pull Request to the Mobile Application Security Verification Standard. Please make sure that:

- [x] Your PR is linked to an issue.
- [x] Your PR follows the [contribution guidelines](https://github.com/OWASP/owasp-masvs/blob/master/CONTRIBUTING.md "Contribution guidelines")
- [x] All links are valid and use HTTPS and have been formatted as [TEXT](URL “NAME”)
- [x] In case of a translation: have you made sure that the translation from EN is as close to the EN document as possible?

This PR covers issue #318.

If your pull-request has a focus of rewriting a requirement and/or adding a new one, please make sure you have a backing issue in which your change has been discussed and agreed. If there is none, then your PR will be closed without further review.

### CODE REVIEW

File: Document-ja/0x03-Using_the_MASVS.md [hunk 1]
```diff
@@ -31,7 +33,7 @@ MASVS-L1 を実現するモバイルアプリはモバイルアプリケーシ
 
 #### MASVS-L2: 多層防御
 
-MASVS-L2 は標準要件を超える高度なセキュリティコントロールを導入します。L2 を満たすためには、脅威モデルが存在する必要があり、セキュリティはアプリのアーキテクチャおよび設計の不可欠な部分である必要があります。このレベルはモバイルバンキングなどの機密データを処理するアプリケーションに適しています。
+MASVS-L2 は標準要件を超える高度なセキュリティコントロールを導入します。MASVS-L2 を満たすためには、脅威モデルが存在する必要があり、セキュリティはアプリのアーキテクチャおよび設計の不可欠な部分である必要があります。脅威モデルに基づいて、適切な MASVS-L2 コントールが選択され、正しく実装されているはずです。このレベルはモバイルバンキングアプリなどの機密性の高いデータを処理するアプリに適しています。
```
Comment (okdt) [inline Document-ja/0x03-Using_the_MASVS.md]: s/コントール/コントロール/

**Comment by coky-t** [comment]
I fixed a misspelling. Please check it.

**Comment by sushi2k** [comment]
Thanks guys! @okdt can you check again and approve?

**Comment by commjoen** [comment]
Thank you very much for your hard work @coky-t  & @okdt !

**Pull request merged**